numpy/lib/function_base.py

```diff
@@ -1501,14 +1506,16 @@ def unwrap(p, discont=pi, axis=-1):
     array([ 0.        ,  0.78539816,  1.57079633, -0.78539816,  0.        ]) # may vary
```

Comment:
An example here with integers would be good

Reply:
how about something like this
```
>>> unwrap([0, 1, 2, -1, 0], min_val=-1, max_val=3)
array([0., 1., 2., 3., 4.]) # may vary
```

Reply:
You shouldn't need `may vary`, because it doesn't use floating point printing

Reply:
Please add that example

Reply:
ping